src/auth/app-load/RedirectFromOAuth.ts

```diff
@@ -1,13 +1,15 @@
 import { UserManager } from 'oidc-client-ts';
+import React from 'react';
 import { div, img } from 'react-hyperscript-helpers';
 import { getOidcConfig } from 'src/auth/oidc-broker';
+import { getCurrentLocation } from 'src/libs/nav/location-utils';
 import { useOnMount } from 'src/libs/react-utils';
 
-const RedirectFromOAuth = () => {
+export const RedirectFromOAuth: React.FC = (): React.ReactNode => {
   const userManager: UserManager = new UserManager(getOidcConfig());
 
-  const url = window.location.href;
-  const isSilent = window.location.pathname.startsWith('/redirect-from-oauth-silent');
+  const url = getCurrentLocation().href;
```

Comment:
window.location is difficult to mock in unit tests, and it's a global (awkward actor), which makes async unit tests unworkable.  So this is being shifted to a thin wrapper that is much easier to mock.